pkg/plugins/plugins.go

```diff
@@ -20,6 +20,7 @@ import (
 	"github.com/grafana/grafana/pkg/plugins/backendplugin/secretsmanagerplugin"
 	"github.com/grafana/grafana/pkg/plugins/log"
 	"github.com/grafana/grafana/pkg/plugins/pfs"
+	"github.com/grafana/grafana/pkg/services/accesscontrol"
```

Comment:
We try to keep pkg/plugins free from other grafana dependencies. Can this logic be implemented elsewhere?

Reply:
Sure, makes sense - I moved the logic here 95f21477a135c13997397c3031f197f8c563d02e

I had placed here it to avoid newing an Evaluator each time, but it is likely not necessary.